HakawaiDemo/HakawaiDemo/MentionsDemoViewController.m

```diff
@@ -53,10 +53,6 @@ - (void)viewDidLoad {
                                                                            controlCharacters:controlCharacters
                                                                                 searchLength:3];
 
-        // NOTE: If you want to see an example of a custom chooser, uncomment the following line.
-//        mentionsPlugin.chooserViewClass = [SimpleChooserView class];
```

Comment:
should we just add a customerChooserView option for tests? this should still be tested right? maybe add this to a hammer-testing jira?